modules/hostap/src/supp_events.c

```diff
@@ -205,6 +234,14 @@ int send_wifi_mgmt_conn_event(void *ctx, int status_code)
 
 	if (wpa_s->current_ssid->mode == WPAS_MODE_AP) {
 		event = NET_EVENT_WIFI_CMD_AP_ENABLE_RESULT;
+#ifdef CONFIG_WPA_SUPP_AP
+		if ((status == WLAN_STATUS_SUCCESS) && wpa_s->ap_iface) {
+			if (config_ap_param_skip_inactivity_poll(wpa_s)) {
+				wpa_printf(MSG_ERROR,
+					   "Unable to configure skip_inactivity_poll parameter");
+			}
```

Comment:
If the failure is being ignored, this should be a warning and also please log the ret code for failure.